### ISSUE TITLE
omf search is broken

### ISSUE THREAD

**Issue by assumptionsoup** [oh-my-fish/oh-my-fish#337]
Running on Ubuntu 16.04 with agnoster theme.
```
>>> fish --version
fish, version 2.2.0

>>> omf --version
Oh My Fish! 2

>>> omf update
Oh My Fish is up to date.
agnoster is already up-to-date.
default is already up-to-date.

>>> omf search
Usage: omf search ([-t | --theme]) ([-pkg | --package]) item

>>> omf search node
omf.search.pkg: command not found
~/.local/share/omf/pkg/omf/functions/cli/omf.cli.search.fish (line 3):       omf.search.pkg $argv[1]
                                                                             ^
in function “omf.cli.search”
        called on line 68 of file ~/.local/share/omf/pkg/omf/functions/omf.fish
        with parameter list “node”

in function “omf”
        called on standard input
        with parameter list “search node”


omf.search.theme: command not found
~/.local/share/omf/pkg/omf/functions/cli/omf.cli.search.fish (line 5):       omf.search.theme $argv[1]
                                                                             ^
in function “omf.cli.search”
        called on line 68 of file ~/.local/share/omf/pkg/omf/functions/omf.fish
        with parameter list “node”

in function “omf”
        called on standard input
        with parameter list “search node”

>>> omf search -pkg node
omf.search.pkg: command not found
~/.local/share/omf/pkg/omf/functions/cli/omf.cli.search.fish (line 9):           omf.search.pkg $argv[2]
                                                                                 ^
in function “omf.cli.search”
        called on line 68 of file ~/.local/share/omf/pkg/omf/functions/omf.fish
        with parameter list “-pkg node”

in function “omf”
        called on standard input
        with parameter list “search -pkg node”
```

**Comment by derekstavis** [comment]
Hey! Thanks for reporting! This issue have been fixed in #335. Please issue an `omf update`.

**Comment by assumptionsoup** [comment]
Thanks @derekstavis .  That was weird.  I ran `omf update` again and it told me again that omf was up to date.  So I tried `omf search` again and now it's working!  I guess omf must have updated itself at some point.
